device/barefoot/x86_64-accton_as9516_32d-r0/newport/buffers_defaults_t1.j2

```diff
@@ -14,13 +14,17 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
+            {%- if dynamic_mode is not defined %}
             "size": "{{ ingress_lossless_pool_size }}",
+            {%- endif %}
             "type": "ingress",
             "mode": "dynamic",
-            "xoff": "36222208"
+            "xoff": "3153920"
```

Comment:
Hi
From this PR I see the buffer pool sizes (`ingress_lossless_pool_size`, `ingress_lossy_pool_size`, and `egress_lossy_pool`) are dynamically calculated and the shared headroom pool size (`ingress_lossless_pool.xoff`) is a fixed number.
But in PR Azure/sonic-swss#2317 and Azure/sonic-swss#2323, I see the buffer pool sizes are fixed and the shared headroom pool size is dynamically calculated.
These result in both the buffer pool sizes and the shared headroom pool size are static. Is this a mistake or by intention?
If you are going to have the buffer pool sizes static and the shared headroom pool size dynamic, I would like to suggest
- always defining `size`s in all pools regardless of whether `dynamic_mode` is defined
- defining `xoff` in `ingress_lossless_pool` only if `dynamic_mode` is defined